netbox/templates/dcim/device.html

```diff
@@ -31,9 +31,17 @@ <h5 class="card-header">{% trans "Device" %}</h5>
                         <td class="d-flex justify-content-between align-items-start">
                             {% if object.rack %}
                                 {{ object.rack|linkify }}
-                                <a href="{{ object.rack.get_absolute_url }}?device={{ object.pk }}" class="btn btn-primary btn-sm d-print-none" title="{% trans "Highlight device in rack" %}">
-                                  <i class="mdi mdi-view-day-outline"></i>
-                                </a>
+                                {% if object.parent_bay %}
+                                    {% with object.parent_bay.device as parent %}
+                                        <a href="{{ object.rack.get_absolute_url }}?device={{ parent.pk }}" class="btn btn-primary btn-sm d-print-none" title="{% trans "Highlight device in rack" %}">
+                                          <i class="mdi mdi-view-day-outline"></i>
+                                        </a>
+                                    {% endwith %}
+                                {% else %}
+                                    <a href="{{ object.rack.get_absolute_url }}?device={{ object.pk }}" class="btn btn-primary btn-sm d-print-none" title="{% trans "Highlight device in rack" %}">
```

Comment:
Could we just change this part

```
?device={{ object.pk }}
```

to this?

```
?device={{ object.parent_bay.device.pk or object.pk }}
```

It would be preferable to avoid the redundant HTML is possible.